### ISSUE TITLE
Slider with grey background

### ISSUE THREAD

**Issue by Havock94** [Dogfalo/materialize#872]
Hi, I'm trying to add an image slider to my page but all I get is a grey background in both full screen and normal display.
I used the source code and the jQuery init showed on the Materialize examples and images are correctly loaded since i can see them if I inspect the page.

**Comment by tmd-uk** [comment]
Have you checked your file permissions/ownership?

**Comment by Havock94** [comment]
What files are you referring to? If you mean the project files then yes, I'm the creator and owner of them.

**Comment by angelpadilla** [comment]
I'm facing the same problem, I just downloaded the latest version of materialize, and I can't figure out the solution for that grey background

**Comment by tmd-uk** [comment]
Have you both initiated the slider in your js file?

**Comment by Havock94** [comment]
Yes I did.

**Comment by tmd-uk** [comment]
This is part of my init.js file 

```javascript
(function($){
  $(function(){

   // The rest above this is hidden

   $('.slider').slider();

  }); // end of document ready
})(jQuery); // end of jQuery name space
```

**Comment by angelpadilla** [comment]
This is my code:
![screen shot 2015-03-15 at 5 49 05 am](https://cloud.githubusercontent.com/assets/5934386/6655858/13ccaa2e-cad7-11e4-82e8-ae7ff3e573ff.png)
![screen shot 2015-03-15 at 5 48 51 am](https://cloud.githubusercontent.com/assets/5934386/6655859/13cd8fd4-cad7-11e4-8954-9cd663b02d84.png)
![screen shot 2015-03-15 at 5 50 39 am](https://cloud.githubusercontent.com/assets/5934386/6655864/4be00bf4-cad7-11e4-9cef-8a00b4810aa4.png)

**Comment by tmd-uk** [comment]
Copy this and try it again 

```javascript
(function($){
  $(function(){

   // The rest above this is hidden

   $('.slider').slider();

  }); // end of document ready
})(jQuery); // end of jQuery name space
```

**Comment by angelpadilla** [comment]
Mmm I already did that, and the same grey background keeps appearing.  
-.-?

**Comment by tmd-uk** [comment]
Have a look at my example -
```javascript
(function($){
  $(function(){

   // The rest above this is hidden

   $('.slider').slider();

  }); // end of document ready
})(jQuery); // end of jQuery name space
``` 
and your example - 

```javascript

  $(function(){

   $('.slider').slider({full_width: true});

  }); // end of document ready

```

Notice the difference? You have left out this, this will ensure you don't have to add the script tags.  just before the end body tag. See what I mean?-
```javascript
(function($){

   //your code in here

  // End Document Ready
}) (jQuery); // End of jQuery name space

```

**Comment by angelpadilla** [comment]
Yeah I know, I already copy the code exactly as yours, and isn't working, I just started the project, I don't have anything but the code for the slider, I can't see what is wrong.

**Comment by tmd-uk** [comment]
Shall I do a screen record for you to show you I did it?

**Comment by jvv** [comment]
This just happened to me, because I ran .slider() twice. As far as I can see, the slider takes the src attribute and puts this in a css background. The second time, the src attribute is empty(-ish), and can't set this to the background image.

**Comment by AEgan** [comment]
Also ran into this issue, followed the suggestion from @jvv and sure enough `$('.slider').slider()` was being called twice. Fixed the problem for me.

**Comment by Musbell** [comment]
$('.slider').slider() initialization should be added into the materialized css documentation or else someone wouldn't take note of that.

**Comment by jvv** [comment]
Like: http://materializecss.com/media.html?

**Comment by siderisltd** [comment]
Hey I know its been a long time since the question is posted but I've just got the same issue with the gray screen. I was literally copy pasting the example in the materialize library. Then I tried to downgrade the materialize. Just to be clear there are no errors in the console, I was trying to run it on plnkr. In my case I was also referring to jqueryUI which script reference was under materialize. When i moved it on top to load first before materialize script it was fine. Hope it helps to someone because it's pretty fucked up bug :D Correction : And now the jQueryUI which I'm using for animations is not working at all and fails silently
